index.js

```diff
@@ -92,7 +108,14 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+
+  if (movies.length === 0) { throw [] }
+
+  return movies.filter(movies => movies.genre.toLowerCase().split(", ").includes(genre.toLowerCase()))
```

Comment:
Great splitting of the string into an array, here! If you did `.includes` on the string, then a genre of "war" would match a genre of "heartwarming". Well done avoiding that issue!